kclvm/api/src/testdata/get-schema-type-mapping.json

```diff
@@ -1,4 +1,4 @@
 {
 	"file": "schema.k",
-	"code": "schema Person:\n    name: str\n    age: int\n\nperson = Person {\n    name = \"Alice\"\n    age = 18\n}\n"
+	"code": "schema Person:\n    '''Schema documents'''\n    name: str\n    age?: int\n\nperson = Person {\n    name = \"Alice\"\n    age = 18\n}\n"
```

Comment:
Could you please add more test cases that has complete docstring sections(Attributes, Examples) :D

Reply:
> Could you please add more test cases that has complete docstring sections(Attributes, Examples) :D

fixed.